File: src/c/fizzbuzz-3.c
// Author https://stackoverflow.com/users/1016716/mr-lister

#include <stdio.h>

int main(void)
{
   int i = 1;
   for (; i<=1000000; ++i) {
      printf("number= %d %s%s\n", i, i%3?"":"Fizz", i%5?"":"Buzz");
   }
   return 0;
}

File: src/fizzbuzz-3.js
#!/usr/bin/nodejs

// Author: Kevlin Hennery
// taken from FizzBuzz Trek

fizzbuzz = (
    lambda n:
        'Fizz' * (n % 3 == 0) +
        'Buzz' * (n % 5 == 0) +
        str(n) * (n % 3 != 0 and n % 5 != 0))

fizzbuzz
